APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -125,7 +125,11 @@ private func checkPurchasesPurchasingAPI(purchases: Purchases) {
     purchases.purchase(package: pack) { (_: StoreTransaction?, _: CustomerInfo?, _: Error?, _: Bool) in }
     purchases.syncPurchases { (_: CustomerInfo?, _: Error?) in }
 
-    purchases.checkTrialOrIntroductoryPriceEligibility([String]()) { (_: [String: IntroEligibility]) in }
+    purchases.checkTrialOrIntroDiscountEligibility([String]()) { (_: [String: IntroEligibility]) in }
+    purchases.checkPromotionalDiscountEligibility(forProductDiscount: discount,
+                                                  product: storeProduct) { (_, _) in
```

Comment:
Nooo we need the types here. That's our way to ensure the closure parameters don't change.

I know you removed them for the line length. I would just add a line break after `(` to solve that.

Reply:
Done in #1229.